maven-plugin/pom.xml

```diff
@@ -29,6 +29,11 @@
             <groupId>org.apache.maven.shared</groupId>
             <artifactId>maven-verifier</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.vdurmont</groupId>
+            <artifactId>semver4j</artifactId>
+            <version>3.1.0</version>
+        </dependency>
```

Comment:
Could this be implemented using the java-semver library we already have instead of adding a new dependency?

Reply:
True, I've just seen that, lemi use the existing library!

Reply:
done!